ckanext/canada/tables/admin_aircraft.yaml

```diff
@@ -0,0 +1,330 @@
+dataset_type: admin_aircraft
+target_dataset: admin_aircraft
```

Comment:
this would be the first type with an underscore in the name. I've been trying to keep these names brief and without underscores to avoid mixing `-` and `_` in a bunch of places which can be confusing.